packaging/os/yumrepo.py

```diff
@@ -0,0 +1,501 @@
+#!/usr/bin/python
+# encoding: utf-8
+
+# (c) 2015, Jiri Tyr <jiri.tyr@gmail.com>
+#
+# This file is part of Ansible
+#
+# Ansible is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# Ansible is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with Ansible.  If not, see <http://www.gnu.org/licenses/>.
+
+
+from ansible.constants import get_config
+from ansible.constants import load_config_file
+from ansible.module_utils.basic import *
+from ansible.module_utils.urls import *
```

Comment:
These should appear at the bottom.  One reason for that is to maintain useful line number information in tracebacks (see the above comment for a description of the textual preprocessing taking place; if the contents of those module utils get included here, all the line numbers below will shift).

This is mentioned in the [Ansible module checklist](http://docs.ansible.com/ansible/developing_modules.html#module-checklist) BTW.